pubsub-adapter/src/main/kotlin/io/ably/lib/realtime/RealtimeChannelAdapter.kt

```diff
@@ -0,0 +1,53 @@
+package io.ably.lib.realtime
+
+import com.ably.pubsub.RealtimeChannel
+import com.ably.pubsub.RealtimePresence
+import io.ably.lib.types.*
+
+internal class RealtimeChannelAdapter(private val legacyChannel: Channel) : RealtimeChannel {
```

Comment:
Does `legacy` give the wrong vibe here? We're not really dealing with a "channel that isn't really a channel any more", just a different implementation?

Reply:
happy to rename :) what do you think is better: `underlyingChannel`, `javaChannel`, `unwrappedChannel` maybe something else?